BlueElectrum.js

```diff
@@ -8,11 +8,8 @@ const reverse = require('buffer-reverse');
 const ElectrumClient = require('electrum-client');
 
 const storageKey = 'ELECTRUM_PEERS';
-export const defaultPeer = { host: '188.166.204.85', tcp: '50001' };
-const hardcodedPeers = [
-  { host: '188.166.204.85', tcp: '50001' },
-  { host: '157.245.20.66', tcp: '50001' },
-];
+export const defaultPeer = __DEV__ ? { host: '188.166.204.85', tcp: '50001' } : { host: '188.166.204.85', tcp: '443' };
```

Comment:
why don't we want to use tls on development env?

Reply:
We do. It is nasty workaround. The thing is solve with envs in here https://github.com/bitcoinvault/GoldWallet/pull/247
Once we finally merge develop into master it will be ok

Reply:
if it is for tests purpose it might not work in test_flow pipeline as it the env is not set to development

Reply:
well, locally it works. And those tests still require some refactoring.